utility/src/main/java/org/oppia/android/util/image/BUILD.bazel

```diff
@@ -0,0 +1,114 @@
+"""
+General purpose utilities for images.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "asset",
+    srcs = [
+        "ImageAssetFetcher.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
Can this one be private?

Reply:
@fsharpasharp it's used in utility build so I'll make it visible to utility. Thanks for bringing it to my notice

Reply:
Done